core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/claim/DefaultClaimManager.java

```diff
@@ -17,80 +17,74 @@
  */
 package org.wso2.carbon.user.core.claim;
 
+import javax.sql.DataSource;
+import javax.xml.stream.XMLStreamException;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.user.api.Claim;
+import org.wso2.carbon.user.api.RealmConfiguration;
+import org.wso2.carbon.user.core.AuthorizationManager;
 import org.wso2.carbon.user.core.UserStoreException;
-import org.wso2.carbon.user.core.claim.builder.ClaimBuilder;
-import org.wso2.carbon.user.core.claim.builder.ClaimBuilderException;
-import org.wso2.carbon.user.core.claim.dao.ClaimDAO;
-import org.wso2.carbon.user.core.internal.UMListenerServiceComponent;
-import org.wso2.carbon.user.core.listener.ClaimManagerListener;
+import org.wso2.carbon.user.core.UserStoreManager;
+import org.wso2.carbon.user.core.config.RealmConfigXMLProcessor;
+import org.wso2.carbon.user.core.dto.ClaimConfig;
+import org.wso2.carbon.user.core.util.FileBasedClaimBuilder;
 
-import javax.sql.DataSource;
+import java.io.IOException;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
-import java.util.Map.Entry;
-import java.util.concurrent.ConcurrentHashMap;
 
 public class DefaultClaimManager implements ClaimManager {
 
-    private ClaimDAO claimDAO = null;
+    private static Log log = LogFactory.getLog(DefaultClaimManager.class);
```

Comment:
Usually the logger instances are made final as one is going to be enough for the entire lifetime of the class.